docs/tokens.md

```diff
@@ -0,0 +1,257 @@
+# **Tokens && Fees**
+
+## **Table of content**
+
+- [**Tokens \&\& Fees**](#tokens--fees)
+  - [**Table of content**](#table-of-content)
+  - [**Tokens**](#tokens)
+    - [**Supported tokens**](#supported-tokens)
+    - [**Listing a token**](#listing-a-token)
+    - [**Tokens not available for fee payment**](#tokens-not-available-for-fee-payment)
+  - [**Fees**](#fees)
+    - [**Fee costs**](#fee-costs)
+    - [**How fees are paid**](#how-fees-are-paid)
+  - [**Transaction batching**](#transaction-batching)
+
+## **Tokens**
+
+RIF Rollup has a list of accepted tokens. A token is added to RIF Rollup via a transaction to its smart contract on L1.
+The listed tokens are the only tokens available to transact or pay the fees on L2. Each token has an ID, the L1 native
+token (RBTC) has the ID zero.
+
+RIF Rollup requires fees for transactions to cover expenses for network operations and paying gas fees on L1. In
+contrast to L1 transactions (though it can be achieved by using [RIF Relay](https://github.com/rsksmart/rif-relay)), L2
+transactions can be paid using any listed token that is available to be used as a fee token. A fee token is a listed
+token with a liquid market price.
+
+### **Supported tokens**
+
+RIF Rollup can be used to transfer RBTC and ERC20 tokens. The full list of currently supported tokens is available under
+this [link](https://explorer.rollup.rifcomputing.net) (opens new window).
```

Comment:
I guess we should explain how to check the currently supported tokens locally. Additionally we could specify the testnet and mainnet links.